hpx/runtime/threads/coroutines/detail/context_linux_x86.hpp

```diff
@@ -140,7 +155,23 @@ namespace hpx { namespace threads { namespace coroutines
 
             x86_linux_context_impl()
                 : m_stack(nullptr)
+#if defined(HPX_HAVE_THREAD_STACKOVERFLOW_DETECTION)
+            {
+                segv_stack.ss_sp = valloc(SEGV_STACK_SIZE);
+                segv_stack.ss_flags = 0;
+                segv_stack.ss_size = SEGV_STACK_SIZE;
+
+                bzero(&action, sizeof(action));
```

Comment:
May I suggest to use `std::memset(&action), '\0', sizeof());` instead of `bzero` (also `#include <cstring>` for memset to be declared)? `bzero` is not portable (not that we would need for it to be portable) and not part of the std C++ library.